python/ray/util/dask/optimizations.py

```diff
@@ -1,167 +0,0 @@
-import operator
-import warnings
-
-import dask
-from dask import core
-from dask.core import istask
-from dask.dataframe.core import _concat
-from dask.dataframe.optimize import optimize
-from dask.dataframe.shuffle import shuffle_group
-from dask.highlevelgraph import HighLevelGraph
-
-from .scheduler import MultipleReturnFunc, multiple_return_get
-
-try:
-    from dask.dataframe.shuffle import SimpleShuffleLayer
-except ImportError:
-    # SimpleShuffleLayer doesn't exist in this version of Dask.
-    SimpleShuffleLayer = None
-
-if SimpleShuffleLayer is not None:
```

Comment:
SimpleShuffleLayer has been removed in 2025.1.0.

Reply:
This might be reimplemented using SimpleShuffle as in dask/dataframe/dask_expr/_shuffle.py